test/src/main/scala/scala/test/scala_import/ScalaImportExposesFileJarsTest.scala

```diff
@@ -0,0 +1,23 @@
+package scala.test.scala_import
+
+import com.google.common.cache.Cache
+import org.apache.commons.lang3.ArrayUtils
+import org.specs2.mutable.SpecificationWithJUnit
+import com.lucidchart.relate.SqlRow
+
+class ScalaImportExposesJarsTest extends SpecificationWithJUnit {
+
+  "scala_import" should {
+    "enable using the jars it exposes" in {
```

Comment:
Do we need this here? I think it was copied from another test and wasn’t removed right?